homeassistant/components/vasttrafik/sensor.py

```diff
@@ -107,7 +107,7 @@ def update(self):
             self._departureboard = self._planner.departureboard(
                 self._departure['id'],
                 direction=self._heading['id'] if self._heading else None,
-                date=datetime.now()+self._delay)
+                date=now(self.hass.config.time_zone)+self._delay)
```

Comment:
`now()` is already in the config zone specified by `hass` 👍

Reply:
I have tried to figure out this, testing multiple variations without luck.
Do you have a reference I can look at?

Reply:
This should work:

```suggestion
                date=now()+self._delay)
```